lib/streamlit/hashing.py

```diff
@@ -290,8 +294,11 @@ def _to_bytes(self, obj, context):
                     return self.to_bytes("%s.%s" % (obj.__module__, obj.__name__))
 
                 h = hashlib.new(self.name)
-                # TODO: This may be too restrictive for libraries in development.
```

Comment:
If this is an actual limitations, we should link an issue to it. Since I am changing this code, I am removing it.

Reply:
Agreed. 

I don't think this is a use-case that matters for Streamlit authors anyway.